packages/paste-website/src/pages/utilities/flex/index.mdx

```diff
@@ -0,0 +1,340 @@
+---
+title: Flex - Utilities
+description: Flex is a utility component that allows engineers to build flexbox based elements and layouts. These flexible elements and layouts allow your components to adapt to the available space or screen size.
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Flex} from '@twilio-paste/flex';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import Changelog from '@twilio-paste/flex/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteUtility(filter: {name: {eq: "@twilio-paste/flex"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/flex/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Flex"
+  categoryRoute={SidebarCategoryRoutes.UTILITIES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/utilities/flex"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/utilities-flex--flex-alignment-options"
+  data={props.data.allPasteUtility.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+### About Flex
+
+Flex utilizes a simplified flexbox API that allow engineers more efficient ways to layout, align, and
+distribute space among elements in a container. The Flex API helps take the confusion out of using
+flexbox CSS properties, by giving engineers simple methods to change properties like flex,
+flex-direction, flex-wrap, and so on.
+
+Flexible elements can adapt to fill space within the layout. This allows Flex to work well for
+product layouts or elements that change orientation, resize, stretch, or shrink.
+
+The Flex API was created because flexbox can be a difficult CSS property to wrap your head around,
+which leads to confusion on how it all works. If you’d like to learn more about flexbox, the
+learning resources below provide further information:
+
+- [MDN Learn Flexbox](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox)
+- [CSS Tricks Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
+
+#### Accessibility
+
+When using Flex, one must keep in mind the reading order of the elements. This helps people with
```

Comment:
We don't expose flex order, right? So if you can't override it or specifiy it, should we even mention this?

Reply:
Right. We're not exposing order so this could be dropped.

Reply:
I've swapped out with similar copy to `Box`.